draft/2020-12/release-notes.md

```diff
@@ -102,3 +102,85 @@ Here are some examples to illustrate the changes.
     </td>
   </tr>
 </table>
+
+## $dynamicRef and $dynamicAnchor
+The `$recursiveRef` and `$recursiveAnchor` keywords were replaced by the more
+powerful `$dynamicRef` and `$dynamicAnchor` keywords. `$recursiveRef` and
+`$recursiveAnchor` were introduced in the previous draft to solve the problem of
+extending recursive schemas. As the "recursive" keywords got some use and we
+understood them better, we discovered how we could generalize them to solve even
+more types of problems. The name change reflects that these keywords are useful
+for more than just extending recursive schemas.
+
+A `$dynamicAnchor` can be thought of like a normal `$anchor` except that it can
+be referenced across schemas rather than just in the schema it was defined in.
+You can think of the old `$recursiveAnchor` as working the same way except that
+it only allowed you to create an anchor at the root of the schema and the anchor
+name is always empty.
+
```

Comment:
Another side effect of using named anchors is that you can have more than one in the same schema (and perhaps overlapping). The tree example could be extended to allow for two different types of branches, each of which is recursive and each references the other.

Reply:
Good point. That's worth mentioning, but I'd rather keep the example simple. I don't what this to become a tutorial and every time I attempted to give a more complex example, it kept snowballing in that direction. The link to this page is in a section called "Migrating from older drafts", so I think it's reasonable to limit the examples to how to convert a 2019-09 schema to a 2020-12 schema.